.ci/magic-modules/get_downstream_prs.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+import functools
+import os
+import re
+import sys
+from github import Github
+
+if __name__ == '__main__':
+  g = Github(os.environ.get('GH_TOKEN'))
+  all_prs = functools.reduce(lambda x,y: x+re.findall(r'\ndepends: (https://github.com/.*)', y.body),
```

Comment:
I'm not super familiar with python style, but is this typical? It feels like a lot is being crammed into these two lines.  If it is, I wouldn't mind seeing a quick comment that explains what's going on.